test/elixir/test/partition_all_docs_test.exs

```diff
@@ -6,6 +6,9 @@ defmodule PartitionAllDocsTest do
   Test Partition functionality for for all_docs
   """
 
+  @moduletag :partition
+  @moduletag kind: :cluster
```

Comment:
This should be `:single_node`. Partition feature is not about partitioned cluster it is about [forming logical database  partitions by using a partition key](https://docs.couchdb.org/en/master/partitioned-dbs/index.html).

Reply:
never mind `cluster` is fine.